src/App.js

```diff
@@ -1,11 +1,13 @@
 import "./App.css";
 import { Footer, Navbar, Homepage } from "./Components/Allcomps";
+import { ProductsPage } from "./Components/Products/ProductsPage";
 
```

Comment:
Appreciation: Its great to see that you are using absolute import here . I also use this way to import the files